contrib/ci/run

```diff
@@ -378,6 +378,7 @@ export V=1
 if "$DEPS"; then
     stage install-deps  deps_install
 fi
+stage pep8          find . -name \*.py -exec pep8 {} +
```

Comment:
I would limit pep8 check to `src/tests/intg/` because  SSSDConfig in `src/config` have 380 pep8 violation
And I would like to avoid fixing it ourself. We can create "easyfix" ticket for that